scripts/build/modules.js

```diff
@@ -9,6 +9,7 @@ const { UNIVERSAL } = require("./bundles").bundleTypes
 const HAS_NO_SIDE_EFFECTS_ON_IMPORT = false
 
 const importSideEffects = Object.freeze({
+  "react": HAS_NO_SIDE_EFFECTS_ON_IMPORT,
```

Comment:
I had to do this in order to get CircleCI to pass; otherwise it was [failing](https://circleci.com/gh/lingui/js-lingui/4340) with this message:

```
Error: An external module "react" is used in a DEV-only code path but we do not know if it is safe to omit an unused require() to it in production. Please add it to the `importSideEffects` list in `scripts/rollup/modules.js`.
 ```

I don't _think_ importing `react` has any side effects on import, so I _think_ this should be OK?  Though honestly I was still confused because it seemed like the `react` module should always be imported regardless of whether we're in production or development, since we need to call `React.createElement()` to process any of the JSX, but maybe JSX is processed later in the pipeline. 🤷‍♂